src/adapters/query_context.js

```diff
@@ -0,0 +1,26 @@
+// Used to provide information when an item from autocomplete is selected.
+//
+// The goal is to improve results provided by bragi based on users' selections.
+export default class QueryContext {
+  constructor(term, ranking, lang = null, position = {}) {
+    this.term = term;
+    this.ranking = ranking;
+    this.lang = lang;
+    // `position` field is supposed to contain `lat`, `lon` and `zoom`.
+    this.position = position;
+  }
+
+  fillHeaders(headers) {
+    if (this.position.lat !== undefined &&
+        this.position.lon !== undefined &&
+        this.position.zoom !== undefined) {
+      const { lat, lon, zoom } = this.position;
+      headers['X-QwantMaps-FocusPosition'] = `${lat};${lon};${zoom}`;
```

Comment:
`y;x;z` is a weird convention,
why not `x;y;z` ?
or `z;y;x` to be consistent with the `#map` hash value ?

And the values should be formatted with a fixed number of decimals.